pkg/ovs/ovn-nb-bfd.go

```diff
@@ -3,13 +3,18 @@ package ovs
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"k8s.io/klog/v2"
 
 	"github.com/kubeovn/kube-ovn/pkg/ovsdb/ovnnb"
+	"github.com/kubeovn/kube-ovn/pkg/util"
+
+	"github.com/ovn-org/libovsdb/cache"
+	"github.com/ovn-org/libovsdb/model"
 )
 
-func (c *OVNNbClient) ListBFD(lrpName, dstIP string) ([]ovnnb.BFD, error) {
+func (c *OVNNbClient) ListBFD(lrpName, dstIP string) (*[]ovnnb.BFD, error) {
```

Comment:
why use a pointer to a slice here

Reply:
I used to think that a pointer always has a better performance, but not always actually. i will recover it.

Reply:
fixed